1-Draft/RFCxxxx-PSModulePath.md

```diff
@@ -0,0 +1,159 @@
+---
+RFC:
+Author: Steve Lee
+Status: Draft
+SupercededBy: n/a
+Version: 1.0
+Area: Engine
+Comments Due: 11/30
+Plan to implement: Yes
+---
+
+# PSModulePath When Starting PowerShell within PowerShell
+
+When starting PowerShell from a different version of PowerShell, the PSModulePath
```

Comment:
should probably be explicit about "Windows PowerShell", unless you are including that behavior here. All of the examples are starting Windows PowerShell

Reply:
This applies to both directions